### ISSUE TITLE
ACC-263.1, (Level A)_ Heading _ Ensure heading level matches the heading's visual importance level 

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3269]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Heading
Affected Check Point:	ACC-263.1
Short text:	ACC-263.1, (Level A)_ Heading _ Ensure heading level matches the heading's visual importance level 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Select +Upload YAML.
•	Copy the following code and paste it into the editor: 

apiVersion: v1
kind: Namespace
metadata:
  name: nginx-ns
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment
  namespace: nginx-ns
  labels:
    app: nginx
spec:
  replicas: 2
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
•	Select Upload
•	Select Close
•	Explore the Monitoring and Health section of the Cluster Details page.
•	In the Events section, select All Messages.
•	Select the sorting icon.
•	In the Sort By section, select Last seen.
•	Select OK.

3. Issue Type: Manual
Observed Behavior:
Th Heading level 4 provided for the ‘Uploaded resources’ creates an inappropriate jump in heading levels within the document heading hierarchy
Screenshot:
 
Expected Behavior:
Users should order heading elements properly. The heading element level must match the level communicated visually by the headings on the page. Headings can be used to communicate structure as well as important content although their purpose should generally be to communicate the structure of content. For example, in HTML, h2 elements should generally follow h1 elements, h3 elements should follow h2 elements, etc. Content developers should generally not "skip" levels (e.g. h1 directly to h3). When levels are skipped users of assistive technology may not understand the relationship of content.
Note: This best practice does not say that important text should be made into headings -- it refers to content that is appropriately marked up as a heading already.
While The HTML5 specification allows for the creation of a document outline structure and heading numbers by using structural elements like aside, article, section, nav with heading elements such as h1 this technique is not supported by assistive technology and is not supported in the accessibility API of user agents.

[ACC-263.1, (Level A)_ Heading _ Ensure heading level matches the heading's visual importance level.docx](https://github.com/user-attachments/files/16727245/ACC-263.1.Level.A._.Heading._.Ensure.heading.level.matches.the.heading.s.visual.importance.level.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16727246/Issue.processing-09.2021.2.docx)
